tests/fv/critical_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fv_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	extensions "k8s.io/client-go/pkg/apis/extensions/v1beta1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/projectcalico/k8s-policy/tests/testutils"
+
+	"github.com/projectcalico/felix/fv/containers"
+	"github.com/projectcalico/k8s-policy/tests/testutils"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	"github.com/projectcalico/libcalico-go/lib/client"
+)
+
+var _ = Describe("[Critical] PolicyController", func() {
```

Comment:
Hm, I think `Critical` might not be the right flag for these tests.  It's not clear to me what that means (all of the tests are "critical" in that if they fail then that indicates a bug).

Is the intention that this section includes tests which check for failure scenarios / recovery?

Perhaps a better tag is `[Resilience]` since it tests the controller's resilience to failure scenarios?